physical_infra_provider_configuration/automation/topics/copy_lenovo_xclarity_class.adoc

```diff
@@ -0,0 +1,9 @@
+==== Copy the &nbsp;LenovoXclarity class to the domain created
```

Comment:
It just seems to be adding an extra space, but you may want to remove _&nbsp_ .

Reply:
done